### DISCUSSION TITLE
Add a note that self-hosted is a stable, long-term release

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1402] [Self-Hosted Support]
pretty much daily we get people asking about latest features that are not yet available in the self-hosted version. despite us making it clear that self-hosted is a long term release in many places, many people don't seem to read that. would be nice if we can add a prominent and persistent note on the top of the self-hosted settings/accounts page to say something like this:

_Self-hosted a long term release so latest features won’t be immediately available as they’re battled-tested in the cloud before released to self-hosted. You can find the latest self-hosted releases and features they include here https://github.com/plausible/analytics/releases_

**Comment by ukutaht** [comment]
Also would be helpful to version our documentation so people on 1.3.0 would only see documentation that was relevant when that version went live.

**Reply by metmarkosaric** [reply]
that could be nice! is there a way to make a connection between Docusaurus and self-hosted versions? wasn't aware of it

**Reply by ukutaht** [reply]
I think it can be done but it has to be managed manually: https://docusaurus.io/docs/versioning

**Comment by blox814123** [comment]
Is there any ETA on the next stable-release update?

**Reply by metmarkosaric** [reply]
we don't have any ETAs. as soon as it's ready it will be on github
